src/components/schedule-page/validation-drawer/error-list.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ScheduleErrorMessageModel } from "../../../common-models/schedule-error-message.model";
+import ErrorListItem from "./error-list-item.component";
+
+interface Options {
+  errors?: ScheduleErrorMessageModel[];
+}
+
+export default function ErrorList({ errors = [] }: Options): JSX.Element {
+  return (
+    <div>
+      {errors?.length > 0 &&
+        errors.map(
+          (error, index): JSX.Element => (
+            <ErrorListItem key={error.toString().substr(2, 9) + index} error={error} />
```

Comment:
W tym przypadku taki sam efekt miałoby kluczowanie po indeksie (co jest złe) *-pisałam wyżej w odpowiedzi do review-* więc co z tym zrobić? Miałam pomysł na nakładkowy interfejs do ScheduleErrorMessageModel dodający każdemu błędowi property id. To ma sens?

Reply:
Tak, masz rację.
Źle ci napisałem: chodziło o to, żeby brać pierwsze litery z wiadomości która jest dolączona do błedu.
Czyli `error.message.substr(2,9)` + index
A `index` jest potrzebny do tego, żeby handlować kilka wiadomości które idą po sobie z tym samym kodem

Reply:
ok, zmieniłam 😄